src/set_theory/ordinal_notation.lean

```diff
@@ -5,10 +5,117 @@ Author: Mario Carneiro
 
 Ordinal notations (constructive ordinal arithmetic for ordinals < ε₀).
```

Comment:
```suggestion
```

Reply:
Not necessary any more since I have moved back Cantor normal form to another file.